web-sdk/v3.md

```diff
@@ -151,7 +161,27 @@ const label = {
   <tr>
     <td><code>callback</code></td>
     <td>function</td>
-    <td>Function to be called when the form is submitted.</td>
+    <td><b>Deprecated</b> Function to be called when the card enroll succeeds or fails.</td>
+  </tr>
+  <tr>
+    <td><code>onCardEnrolledCallback</code></td>
+    <td>function</td>
+    <td>Function to be called when a Card is enrolled successfuly. Receives the Card entity as argument.</td>
```

Comment:
Do you think put the link to the card entity docs here could be a good idea to start with?
What do you think @gabegodoi182 @cornelfd @murtas ?

Reply:
I would say no. https://fidelapi.com/docs/cards/ actually presents the API representation (list of items) and includes some deprecated fields which we are discontinuing in the SDK (API should follow).

Reply:
Yeah you're right.  I think we should create a task to improve this aspect of the documentation.

Reply:
@murtas Couldn't we just put an example here in the `.md` of the objects that customers can receive in the callbacks? Later we can use JDocs, do the necessary research and do something fancier.

Reply:
What concerns me the most is that the object is quite extensive (14 properties) for the CardEntity itself. Plus the remaining payloads, which include errors with their types.

I'm expecting that level of detail to come from the library documentation (JSDoc). But if you think that it will end up in this document anyway I can try to come up with new sections for that.